public/locales/es/translation.json

```diff
@@ -180,21 +181,167 @@
   "tab5-blockquote1": "Para obtener más información, visite <2>Disposiciones de la Ley CARES</2>.",
   "page-not-found-header": "Page Not Found",
   "page-not-found-text": "The page or form you requested cannot be found. If your link worked before but does not work now, then this page has moved or was deleted as part of our ongoing efforts to make our website work better for you.",
+  "yesnoquestion": {
+    "Yes": "Yes",
+    "No": "No"
+  },  
   "retrocert-login": {
-    "title": "Retroactive Unemployment Certification",
-    "subheader": "Help us find you",
-    "help-text": "We need to make sure your identity matches EDD records so that we show you accurate records.",
-    "last-name-label": "Last name",
-    "ssn-label": "Social Security Number",
+    "title": "Retroactive Certification for Unemployment",
```

Comment:
We could move the title out of the pages since it's the same on all the pages. Not a big deal either way.

Reply:
Thank you for pointing this out! The title is actually supposed to be different on the question pages! I'll update now.